src/tests/unit/tests/injected/components/command-bar.test.tsx

```diff
@@ -139,13 +151,11 @@ describe('CommandBar', () => {
                 onClickCopyIssueDetailsButton: onClickMock.object,
             };
 
-            const wrapper = shallow(<CommandBar {...props} />);
-
-            const button = wrapper.find(CopyIssueDetailsButton);
+            render(<CommandBar {...props} />);
 
-            button.prop('onClick')(eventStub);
+            getMockComponentClassPropsForCall(CopyIssueDetailsButton).onClick();
```

Comment:
We should call the `onClick` with the `eventStub` so we can make sure that the mock is called properly.
```suggestion
            getMockComponentClassPropsForCall(CopyIssueDetailsButton).onClick(eventStub);
```